### DISCUSSION TITLE
Post Install Script to Upgrade to V8 requires V8 to run

### DISCUSSION THREAD

**Discussion by pacmac** [tteck/Proxmox#3221] [General] [answered]
I understand that the post install script can be used to upgrade version 7 to 8, and that as from July (next month) that V8 is required.

However when I run the script to upgrade to V8 I get:

```
Start the Proxmox VE Post Install Script (y/n)?y
 ✗ This version of Proxmox Virtual Environment is not supported
Requires Proxmox Virtual Environment Version 8.0 or later.
```

what am I doing wrong ?

**Comment by pacmac** [comment]
Should I just run the https://raw.githubusercontent.com/tteck/Proxmox/main/misc/pve8-upgrade.sh instead ?

**Comment by Mellowlynx** [comment, marked as answer]
First, utilize the command `pve7to8 --full` to check readiness for the transition.

This command is designed to transition a currently running PVE 7 instance into a running PVE 8 instance.

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/pve8-upgrade.sh)"`

To ensure a proper source installation, please respond with "yes" to all questions prompted during the installation process.

**Comment by pacmac** [comment]
Thanks for the quick response, yes I have already run the `pve7to8 --full` command and fixed a few issues, and also stopped all instances before upgrading.

The script ran but appeared to fail at the last step with NGINX errors:

```
Running hook script 'zz-proxmox-boot'..
Re-executing '/etc/kernel/postinst.d/zz-proxmox-boot' in new private mount namespace..
No /etc/kernel/proxmox-boot-uuids found, skipping ESP sync.
Processing triggers for pve-ha-manager (4.0.4) ...
Errors were encountered while processing:
 nginx
 libnginx-mod-http-image-filter
 libnginx-mod-stream
 libnginx-mod-stream-geoip
 libnginx-mod-http-xslt-filter
 libnginx-mod-http-geoip
 nginx-core
 libnginx-mod-mail
Removing subscription nag from UI...
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Not sure yet whether these are critical or what the implications are.

**Comment by pacmac** [comment]
OK, found the issue, it was a bad config entry for a nginx proxy that no longer existed.

All is good

Upgraded 2 servers, thank you.

**Reply by Mellowlynx** [reply]
I was about to look it up, but great that you found the problem.
_Could you select my first replay as the answer, please? (It will be closed as Answered, instead of unanswered :) )_

**Comment by pacmac** [comment]
OK, I do have now several issues.

NUT server is no longer working and cannot communicate with the UPS over USB even though lsusb shows the UPS, and it was working before upgrade.

Some kind of network issue where dns now returns IPV6 ip addresses and this is causing all requests to fail, i.e system updates, pings etc all now fail.

**Comment by pacmac** [comment]
Hmm, this now affects all of my containers, now none of them can access the internet due to IPV6 dns lookups which wont resoilve.

**Reply by tteck** [reply]
Disable IPv6 in your router.

**Comment by pacmac** [comment]
I am working through the issues, disabled DHCPV6 service on opnsense yet containers are still getting an IPV6 IP, no idea why even after reboot of opnsense and proxmox host and containers.

Also, frigate crashed and was unable to passthrough GPU see here: https://github.com/blakeblackshear/frigate/issues/9574

```
RuntimeError: Failed to create plugin /usr/local/lib/python3.9/dist-packages/openvino/libs/libopenvino_intel_gpu_plugin.so for device GPU
```

Solution for me was to change device:  GPU:

```
detectors:
  ov:
    type: openvino
    device: AUTO
```

And on Alpine Linux prevent IPV6:

```
sysctl -w net.ipv6.conf.all.disable_ipv6=1
echo "net.ipv6.conf.all.disable_ipv6 = 1" >>/etc/sysctl.conf
rc-update add sysctl default
sysctl -p
```

and on Debian / Ubuntu:

```
sysctl -w net.ipv6.conf.all.disable_ipv6=1
echo "net.ipv6.conf.all.disable_ipv6 = 1" >>/etc/sysctl.conf
sysctl -p
```

Still debugging others will update when I have solutions.

**Comment by pacmac** [comment]
so far unable to resolve while NUT has stopped communicating with the UPS over usb.

**Comment by pacmac** [comment]
Googled it and lots of people reporting same issue but no solutions that I can find:

```
Jun 17 13:50:44 ellis-vm nut-monitor[1246]: Poll UPS [eatonElipse@localhost] failed - Driver not connected
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51492]: Network UPS Tools - Generic HID driver 0.47 (2.8.0)
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51492]: USB communication driver (libusb 1.0) 0.43
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51492]: libusb1: Could not open any HID devices: insufficient permissions on everything
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51492]: No matching HID UPS found
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51491]: Driver failed to start (exit status=1)
Jun 17 13:50:45 ellis-vm nut-driver@eatonElipse[51491]: Network UPS Tools - UPS driver controller 2.8.0
Jun 17 13:50:45 ellis-vm systemd[1]: nut-driver@eatonElipse.service: Control process exited, code=exited, status=1/FAILURE
```

```
libusb1: Could not open any HID devices: insufficient permissions on everything
```

Any suggestions ?

**Reply by Mellowlynx** [reply]
Maybe this might help:
https://www.reddit.com/r/debian/comments/14kj3bl/comment/jxnk3k9
